webpack.config.dev.js

```diff
@@ -17,7 +17,7 @@ export default {
   target: 'web', // necessary per https://webpack.github.io/docs/testing.html#compile-and-test
   output: {
     path: __dirname + '/dist', // Note: Physical files are only output by the production build task `npm run build`.
-    publicPath: '/',
+    publicPath: 'http://localhost:3000/', // Use absolute paths to avoid the way that URLs are resolved by Chrome when they're parsed from a dynamically loaded CSS blob. Note: Only necessary in Dev.
```

Comment:
I thought for was dynamic, if you are already running you could be on 3002. Wouldn't there be conflicts with this? May want to consider using `process.env.port` to cover that scenario.